features/gebruik_in_lopende_tekst.feature

```diff
@@ -34,138 +34,126 @@ Rule: gebruikInLopendeTekst voor een persoon zonder adellijke titel of predicaat
   - Aanduiding naamgebruik "E" (eigen naam) wordt gehanteerd voor een persoon die geen actuele en geen ontbonden huwelijken/partnerschappen heeft gehad
```

Comment:
Nog een opmerking over regel 32:

`  - De eerste naamcomponent direct na "mevrouw" of "de heer" begint met een hoofdletter`

Dit geldt natuurlijk alleen als het eerste naamcomponent ook begint met een letter.
Daarom deze regel wellicht wijzigen in:

`  - Als de eerste naamcomponent direct na "mevrouw" of "de heer" begint met een letter wordt deze als hoofdletter weergegeven`

Reply:
Aangepast